frontend/package.json

```
@@ -3,106 +3,37 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "autoprefixer": "7.1.6",
-    "babel-core": "6.26.0",
-    "babel-eslint": "7.2.3",
-    "babel-jest": "20.0.3",
-    "babel-loader": "7.1.2",
-    "babel-preset-react-app": "^3.1.2",
-    "babel-runtime": "6.26.0",
-    "case-sensitive-paths-webpack-plugin": "2.1.1",
-    "chalk": "1.1.3",
-    "css-loader": "0.28.7",
-    "dotenv": "4.0.0",
-    "dotenv-expand": "4.2.0",
-    "eslint": "5.6.1",
-    "eslint-config-airbnb": "17.1.0",
-    "eslint-loader": "2.1.1",
-    "eslint-plugin-babel": "5.2.1",
-    "eslint-plugin-import": "2.14.0",
-    "eslint-plugin-jest": "21.23.0",
-    "eslint-plugin-jsx-a11y": "6.1.1",
-    "eslint-plugin-react": "7.11.1",
-    "extract-text-webpack-plugin": "3.0.2",
-    "file-loader": "1.1.5",
-    "fs-extra": "3.0.1",
-    "html-webpack-plugin": "2.29.0",
-    "jest": "23.6.0",
-    "jest-cli": "23.6.0",
-    "object-assign": "4.1.1",
+    "node-sass": "^4.11.0",
     "pinpoint-client": "file:../client",
-    "postcss-flexbugs-fixes": "3.2.0",
-    "postcss-loader": "2.0.8",
-    "promise": "8.0.1",
-    "prop-types": "15.6.2",
-    "raf": "3.4.0",
-    "react": "^16.5.2",
-    "react-dev-utils": "^5.0.2",
-    "react-dom": "^16.5.2",
+    "prop-types": "^15.6.2",
+    "react": "^16.7.0",
```

can we lock all this down to specific versions?

```suggestion
    "react": "16.7.0",
```

---

TODO: tomorrow do this